test/test-genesis.ts

```diff
@@ -384,4 +385,66 @@ describe('Genesis Contract', () => {
       contract.connect(owner).mintReservedGods(whitelistedAddr, 10),
     ).to.be.revertedWith('Not enough reserved gods left');
   });
+
+  it('Contract base URI is unrevealed URI if not changed', async function () {
+    await contract.connect(owner).unpause();
+    expect(await contract.tokenURI(1)).to.be.equal(constants.unrevealedURI);
+  });
+
+  it('Contract base URI is unrevealed URI if not changed', async function () {
```

Comment:
Duplicated test, you can merge them together.
I've put invalid IDs to the test because it shouldn't error, but its the same test.

Reply:
done